ADOPTERS.md

```diff
@@ -0,0 +1,7 @@
+# Who uses GitOps? 
+
+As the GitOps movement grows, we'd like to hear from organizations who are successfully implementing GitOps! Please send a PR with *your* organization name if you are adopting the [GitOps principles](https://opengitops.dev/#principles) in practice. Please include how you are adopting.
```

Comment:
I do like the idea that GitOps can be seen as a kind of movement, but I do agree this prob isn't the place to drop that suggestion 🙂

Here's a GitHub code suggestion you can commit directly if you want. Includes what @chris-short said, with small grammar tweaks to accompany that.

```suggestion
As GitOps adoption grows, we'd like to hear from organizations that are successfully implementing GitOps! Please send a PR with *your* organization name if you are adopting the [GitOps principles](https://opengitops.dev/#principles) in practice. Please include how you are adopting.
```